documentation/src/main/asciidoc/userguide/appendices/Configurations.adoc

```diff
@@ -709,7 +709,7 @@ A configuration value key used to indicate that it is safe to cache.
 `*hibernate.jta.cacheUserTransaction*` (e.g. `true` or `false` (default value))::
 A configuration value key used to indicate that it is safe to cache.
 
-`*hibernate.transaction.flush_before_completion*` (e.g. `true` or `false` (default value))::
+`*hibernate.transaction.flush_before_completion*` (e.g. `true` (default value) or `false` )::
 Causes the session be flushed during the before completion phase of the transaction. If possible, use built-in and automatic session context management instead.
```

Comment:
I think the following would be better

```suggestion
`*hibernate.transaction.flush_before_completion*` (e.g. `true` (Hibernate boot default value) or `false` (JPA boot default value) )::
Causes the session be flushed during the before completion phase of the transaction. If possible, use built-in and automatic session context management instead. When Hibernate is link:https://docs.jboss.org/hibernate/orm/current/quickstart/html_single/#hibernate-gsg-tutorial-basic-test[booted through the native API], the default value is `true`, otherwise `false`.
```